third_party/move/move-compiler/src/parser/syntax.rs

```diff
@@ -1946,7 +1948,7 @@ fn parse_unary_exp(context: &mut Context) -> Result<Exp, Box<Diagnostic>> {
 
 // Parse an expression term optionally followed by a chain of dot or index accesses:
 //      DotOrIndexChain =
-//          <DotOrIndexChain> "." <Identifier> [  "(" Comma<Exp> ")" ]
+//          <DotOrIndexChain> "." <Identifier> [ ["::" "<" Comma<Type> ">"]? <CallArgs> ]?
```

Comment:
What is `CallArgs`, used here?  It looks like you may be referring to the arguments parsed by `parse_call_args()` but the grammar term isn't mentioned in the comment for that function.
Please add this name there so it is consistent.

Reply:
Added.